tests/test_builders/test_build_latex.py

```diff
@@ -2286,6 +2286,21 @@ def test_one_parameter_per_line(app):
     )
 
 
+@pytest.mark.sphinx(
+    'latex',
+    testroot='domain-py-python_maximum_signature_line_length',
+    confoverrides={
+        'python_maximum_signature_line_length': 23,
+        'python_trailing_comma_in_multi_line_signatures': False,
+    },
+)
+def test_one_parameter_per_line_without_trailing_comma(app):
+    app.build(force_all=True)
+    result = (app.outdir / 'projectnamenotset.tex').read_text(encoding='utf8')
+
+    assert r'\sphinxparam{\DUrole{n}{f}}}}' in result
```

Comment:
```suggestion
    assert r'\sphinxparam{\DUrole{n}{f}}\sphinxparamcomma' not in result
    assert r'\sphinxparam{\DUrole{n}{f}}}}' in result
```

With this "not in" test, it's a bit easier to read (I missed that there were more closing brackets in the "in" assertion). At least, readers can compare with the `assert r'\sphinxparam{\DUrole{n}{f}}\sphinxparamcomma' in result` other case.